bittensor/core/async_subtensor.py

```diff
@@ -140,23 +146,41 @@ def __init__(
         archive_endpoints: Optional[list[str]] = None,
         websocket_shutdown_timer: float = 5.0,
     ):
-        """
-        Initializes an instance of the AsyncSubtensor class.
+        """Initializes an AsyncSubtensor instance for blockchain interaction.
 
-        Arguments:
-            network: The network name or type to connect to.
-            config: Configuration object for the AsyncSubtensor instance.
-            log_verbose: Enables or disables verbose logging.
+
+        Args:
+            network: The network name or type to connect to (e.g., "finney", "test"). If None, uses the default network
+                from config.
+            config: Configuration object for the AsyncSubtensor instance. If None, uses the default configuration.
+            log_verbose: Enables or disables verbose logging. Defaults to False.
             fallback_endpoints: List of fallback endpoints to use if default or provided network is not available.
-                Defaults to `None`.
-            retry_forever: Whether to retry forever on connection errors. Defaults to `False`.
-            _mock: Whether this is a mock instance. Mainly just for use in testing.
+                Defaults to None.
+            retry_forever: Whether to retry forever on connection errors. Defaults to False.
+            _mock: Whether this is a mock instance. Mainly for testing purposes. Defaults to False.
```

Comment:
```suggestion
            _mock: Whether this is a mock instance. Mainly for testing purposes. Defaults to `False`.
```
Could you please use the same quotes `` to wrap the default values across whole PR? Or is there a special reason to remove it?